tests/test_tools.py

```diff
@@ -16,109 +16,201 @@ def _setup_dirs(monkeypatch, user_filesystem):
     return home_dir
 
 
-def _run_tests(inputs, expected):
-    args = {"username": inputs[0], "email": inputs[1]}
-    expected_username, expected_email = expected
-    config = get_user_info(args)
+def _run_tests(cli_inputs, expected):
+    user_info = {"username": cli_inputs["cli_username"], "email": cli_inputs["cli_email"]}
+    config = get_user_info(user_info=user_info, skip_config_creation=cli_inputs["skip_config_creation"])
+    expected_username = expected["expected_username"]
+    expected_email = expected["expected_email"]
     assert config.get("username") == expected_username
     assert config.get("email") == expected_email
 
 
 params_user_info_with_home_conf_file = [
```

Comment:
`params_user_info_with_home_conf_file` may not be needed - please see our discussion: https://github.com/diffpy/diffpy.utils/pull/236/files#r1885496247 @yucongalicechen

Reply:
I think this is probably a different test case than that issue..?